### DISCUSSION TITLE
🧳Bigger Backpack Mod

### DISCUSSION THREAD

**Discussion by Superbiabugreport** [vaido-world/7daystodie-dedicated-server#6] [Ideas]
Possibility of having a slightly bigger backpack mod?

**Comment by BoQsc** [comment]
Currently we have 27 slots unlocked by default. 
And 18 slots locked up. **Overall slots being:** 45

<img src="https://user-images.githubusercontent.com/21064622/124137300-37203180-da8e-11eb-8eb7-3d42667779a9.png" alt="drawing" width="600"/>



Personally I think that would be not very 'aesthetic' thing to have in the server.

I'm not really a fan of larger backpack mods. 
But they seem to be popular among other servers. 

Currently I have no plans in doing that, 
unless more server's community members asks for it and suggest to maintain it, 
refine it for the future versions of 7 Days To Die.

But there seems to be a lot to choose from
* https://7daystodiemods.com/mega-backpack/
* https://7daystodiemods.com/bigger-backpack-mod-60-96-slot-with-without-food-water-bar-on-hud/

And many more here:  
https://www.nexusmods.com/7daystodie/search/

So it is easy to implement into the server.

![image](https://user-images.githubusercontent.com/21064622/124134314-5e293400-da8b-11eb-9c20-9b2c40971d9c.png)
